src/app/event/[id]/page.tsx

```diff
@@ -1,72 +1,91 @@
 "use client";
 import React from "react";
 import { ArrowLeft, Clock4, MapPin } from "lucide-react";
-import { useRouter } from "next/router";
+import { useRouter } from "next/navigation";
 import { useEventDetail } from "@/lib/hooks/useEventDetail";
+import { format } from "date-fns";
 
 export default function Page({ params }: { params: { id: string } }) {
-  const { eventDetail, updateEventDetail } = useEventDetail();
-
   const router = useRouter();
+  const { eventDetail } = useEventDetail(params.id);
+
+  const date = new Date();
+  const currentDay = date.getDay();
```

Comment:
why currentDay = new Date().getDay()? Should we get day from eventDetail.startTime/endTime here?